src/basicfuns.jl

```diff
@@ -441,3 +441,72 @@ $(SIGNATURES)
 Compute the complementary double exponential, `1 - exp(-exp(x))`.
 """
 cexpexp(x) = -_expm1(-exp(x))
+
+#=
+this uses the identity:
+
+log(logistic(x)) = -log(1 + exp(-x))
+=#
+"""
+$(SIGNATURES)
+
+The natural logarithm of the `logistic` function, computed more
+carefully and with fewer calls than than the composition
+`log(logistic(x))`.
+
+Its inverse is the [`logitexp`](@ref) function.
+"""
+loglogistic(x::AbstractFloat) = -log1pexp(-x) #
```

Comment:
Why only `AbstractFloat`? I think we could just use
```suggestion
loglogistic(x::Real) = -log1pexp(-x)
```